app/controllers/facility_reviews_controller.rb

```diff
@@ -18,38 +18,42 @@ def new
   end
 
   def create
-    @space = Space.find(params["space_id"])
-    affected_facility_ids = []
+    space_id_param = params.require(:space_id)
+    reviews_and_descriptions = params.require(:space).permit(
+      facility_reviews_attributes: %i[id experience facility_id user_id],
+      space_facilities_attributes: [:facility_id, :description]
+    )
 
-    parsed = parse_facility_reviews(params["facility_reviews"]["reviews"], affected_facility_ids)
+    @space = Space.find(space_id_param)
 
-    unless FacilityReview.create(parsed)
-      render :new, status: :unprocessable_entity
-      return
-    end
+    @affected_facilities = [] # filtered_facility_reviews will mutate this array
```

Comment:
I hate that we have this mutable array here. It was in the old code too, and is a result of facility review aggregation being kind of messed up. Now it's at least tagged as mutable, but I'd love to take the time to refactor it out some other time.